setup.py

```diff
@@ -170,6 +170,7 @@
             'zenodo_frontpage = zenodo.modules.frontpage.views:blueprint',
             'zenodo_search_ui = zenodo.modules.search_ui.views:blueprint',
             'zenodo_theme = zenodo.modules.theme.views:blueprint',
+            'zenodo_deposits = zenodo.modules.deposits.views:blueprint',
```

Comment:
I believe we should have been consistent with module names plurality, thus have ``invenio-deposits`` rather than ``invenio-deposit``, but for now it's deposit thus we should refer to "deposit" whenever we are talking about the module.

Hence it should be ``zenodo_deposit = zenodo.modules.deposit.views:blueprint`` (singular)